test/test_metrics/test_prob_pairwise_measures.py

```
@@ -1,17 +1,114 @@
+    pred[0:30] = 0.2
+    pred[30:50] = 0.45
+    pred[50:80] = 0.65
+    pred[80:] = 0.90
+
+    ppm = PPM(pred, ref)
+    list_sens_exp = [0, 0.4, 1, 1, 1] 
+    list_spec_exp = [1, 1, 1, 0.6, 0]
+    list_ppv_exp = [1, 1, 1, 0.71, 0.5]
+    unique_thresh = [1.9, 0.9, 0.65, 0.45, 0.2]
+
+    t, se, sp, pp, fp = ppm.all_multi_threshold_values()
```

Ruff (0.15.10)

[warning] Unpacked variable `fp` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)